draft-abiggs-saag-key-management-service.md

```diff
@@ -1670,6 +1695,25 @@ JWE(K_ephemeral, {
 })
 ~~~
 
+Buld Delete response message example:
+
+~~~
+JWE(K_ephemeral, {
+{
+  "status": 200,
+  "requestId": "10992782-e096-4fd3-9458-24dca7a92fa5",
+  "authorization": {
+    "uri": "/authorizations",
+    "authIds": {
+      "557ac05d-5751-43b4-a04b-e7eb1499ee0a",
+      "c1047d54-c485-4aa1-9704-fee07adae46c",
+      ...
+    }
+    "resourceUri": "/resources/7f35c3eb-95d6-4558-a7fc-1942e5f03094"
+  }
+})
+~~~
+
```

Comment:
Looks good.

Reply:
We list the `uri` as a way to identify it as authorizations to delete.